exercises/primzahlen.py

```diff
@@ -0,0 +1,26 @@
+from time import *
+prim = [1, ]
+
+
+def main():
+    print("HINWEIS: Die Primzahlen von 1 bis 100'000, mit diesem Code zu berechnen dauert ca. 1079 Sekunden (17:59.147),")
+    print("die Primzahlen von 1 bis 10'000 jedoch brauchen lediglich ca. 11 Sekunden (00:11.274).")
```

Comment:
Merhzeilige Ausgaben können mit 3 Anführungszeichen erzeugt werden - ebenso wie mehrzeilige Kommentare...
```suggestion
    print('''HINWEIS: Die Primzahlen von 1 bis 100'000, mit diesem Code zu berechnen dauert ca. 1079 Sekunden (17:59.147),
die Primzahlen von 1 bis 10'000 jedoch brauchen lediglich ca. 11 Sekunden (00:11.274).'''
    )
```